### PR TITLE
Basic routing setup and header component 

### PR DESCRIPTION
**Pull request by vanshitalavange**
Overview of this PR:
- Mockbee and mockman setup
- basic folder structure 
- basic routing setup 
- header component
- responsive navbar 
- Preview:
-  https://user-images.githubusercontent.com/86543154/159638572-e1bdad2b-1837-494d-813b-91aed3cf3be5.mp4

### CODE REVIEW

**Review by srejitk — APPROVED** [review]
No changes suggested. Liked the approach of using context for navigation.

File: src/components/ResponsiveNavbar/ResponsiveNavbar.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import { Link } from "react-router-dom";
+import { useNavbar } from "../../contexts/navbar-context";
+import "./ResponsiveNavbar.css";
+export function ResponsiveNavbarForMobile() {
+  const { showResponsiveNavbarForMobile, setShowResponsiveNavbarForMobile } =
+    useNavbar();
+  return showResponsiveNavbarForMobile ? (
+    <div className="responsive-navbar flex-column">
+      <button
+        onClick={() => setShowResponsiveNavbarForMobile(false)}
```
Comment (rdshinde) [inline src/components/ResponsiveNavbar/ResponsiveNavbar.jsx:10]: Just a neat one: If you could reduce the name of setter function, I think its too long. for ex. `setResponsiveNavbar`.

**Review by rdshinde — APPROVED** [review]
Just a small suggestion. Otherwise everything is good 👍 .

File: src/components/ResponsiveNavbar/ResponsiveNavbar.jsx (hunk 1 above)
Comment (vanshitalavange) [reply src/components/ResponsiveNavbar/ResponsiveNavbar.jsx:10]: the thing is that I have a responsive navbar for tablet view as well so I wanted to be specific as to what the particular setter function is for

**Pull request merged**